shared/src/sector/registered_proof.rs

```diff
@@ -253,19 +217,19 @@ impl RegisteredSealProof {
         match self {
             Self::StackedDRG64GiBV1
             | Self::StackedDRG64GiBV1P1
-            | Self::StackedDRG64GiBV1P1_Feat_SyntheticPoRep => Ok(StackedDRGWinning64GiBV1),
+            | Self::StackedDRG64GiBV1P1_Feat_SyntheticPoRep => Ok(StackedDRGWindow64GiBV1P1),
```

Comment:
Winning v. Window post?

Reply:
Should be Winning according to the doc comment.

Reply:
Can we consider deleting this method entirely? I'm guessing it's used in Forest, but can we make it a helper method there? The FVM really doesn't care about winning posts.

Reply:
I have nothing against. Let's remove it.